build.gradle.kts

```diff
@@ -19,11 +19,7 @@ repositories {
 
 // Set the JVM language level used to build the project. Use Java 11 for 2020.3+, and Java 17 for 2022.2+.
```

Comment:
The comment also needs to be updated.

This is done to transfer changes from [JetBrains/intellij-platform-plugin-template](https://github.com/JetBrains/intellij-platform-plugin-template) one-to-one.

Reply:
Done